src/test_utils.rs

```diff
@@ -222,16 +224,24 @@ pub mod tests {
                     &unit.file_name,
                     crate::OptimizationLevel::None,
                     debug_level,
+                    crate::OnlineChange::Disabled,
```

Comment:
I think it would make sense to have one function which has OnlineChange enabled for the sake of testing if the section names are generated correctly in the codegen, wdyt?

Reply:
I'll add a test, either now or a followup that has an enabled online change code generated